docs/source/user-guide/python.md

```diff
@@ -103,14 +119,15 @@ The `explain` method can be used to show the logical and physical query plans fo
 The following example demonstrates creating arrays with PyArrow and then creating a Ballista DataFrame.
 
 ```python
-import ballista
+from ballista import Ballista, BallistaBuilder
```

Comment:
Why do we need `Ballista`, looks like everything can be covered with `BallistaBuilder`

Reply:
yep, removing